OrbitQt/CopyableTreeView.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_COPYABLE_TREE_VIEW_H_
+#define ORBIT_QT_COPYABLE_TREE_VIEW_H_
+
+#include <QKeyEvent>
+#include <QTreeView>
+
+class CopyableTreeView : public QTreeView {
```

Comment:
My first thought was this is about the C++-object being copyable, so the name was a little bit misleading to me. But I don't have a good suggestion, so feel free to ignore that comment.

Reply:
Edit: Maybe ClipboardEnabledTreeView or something like that? I'm not sure...

Reply:
Fine with me. Thanks.